openquake/baselib/__init__.py

```diff
@@ -39,14 +38,17 @@ def __getattr__(self, key):
 
 
 config = DotDict()  # global configuration
-if 'VIRTUAL_ENV' in os.environ or hasattr(sys, 'real_prefix'):
```

Comment:
Why you removed `hasattr(sys, 'real_prefix')`

Reply:
Anyway it looks like the best way is https://stackoverflow.com/a/42580137

Reply:
This should solve also the issue when you call directly executables in the venv without loading it first.

Reply:
I undestood `sys.real_prefix` was needed for old versions of virtualenv and not needed anymore.

Reply:
You're right, it works with `virtualenv` but not with `venv`. Anyway https://stackoverflow.com/a/42580137 seems the best solution. But for now I'm fine.